packages/components/addon/components/hds/code-block/copy-button.hbs

```diff
@@ -0,0 +1,13 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Copy::Button
+  class="hds-code-block__copy-button"
+  @text="Copy"
```

Comment:
Do we have a ticket to allow this text to be changed by the consumer?

Reply:
Is there a reason they would need to change it?

Reply:
@KristinLBradley remember how we needed to update the text for our own website, so that the copy buttons could be differentiated from one another in the cases where there was more than one copy button on a page? This would be the same reason to allow the consumer to edit the value here.

Reply:
I added an `aria-describedby` that reads out the associated code to try to address this need. When I tested using a screen reader, it read out the associated code as intended. Is that not a good enough solution? I couldn't modify it as we did for the website copy button as it uses an `Hds::Copy::Button` which itself uses an `Hds::Button` under the hood unlike the website which had its own custom button. (We would have to update the `Hds::Button` itself, although we could look at doing that as a separate update if we need to.)

Reply:
Here's where I initially mentioned this: https://github.com/hashicorp/design-system/pull/1687#issuecomment-1758947613

I discussed the solution with @alex-ju before leaving on vacation. I believe I hid the comment as resolved after discussing with him but I've now unhidden it.

Reply:
@melanie So if we need to do something else for the Copy button to distinguish it (if my `aria-describedby` solution isn't sufficient) would it be best to modify our button component to make it possible to implement a solution similar to what we did for the web docs one or should we just add an additional arg to `CopyBlock` for `@copyButtonText` or whatever we want to call it?

Reply:
I saw your comment about testing it too and I am satisfied with this approach. 👍

Reply:
Great, thanks! @MelSumner

If we need to make later improvements, we can look at doing that as a follow-on. :)